etc/kayobe/ansible/pulp-everything.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Import pulp-repo-sync"
+  import_playbook: pulp-repo-sync.yml
+
+- name: "Import pulp-repo-publish"
+  import_playbook: pulp-repo-publish.yml
+
+- name: "Import pulp-repo-promote-production"
+  import_playbook: pulp-repo-promote-production.yml
```

Comment:
I'm not sure about this one. We should encourage people to only promote when they're going to do upgrades wherever possible.

I think a good balance would be to optionally allow it with a variable, but probably keep the default as false